### PR TITLE
Front end/side navbar

### PR DESCRIPTION
**Pull request by ebauss**

### CODE REVIEW

File: dna-frontend/src/app/core/components/sidebar/sidebar.component.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
+<button
```
Comment (m-a-s-h-e-d) [inline dna-frontend/src/app/core/components/sidebar/sidebar.component.html:1]: - Add classes or ids to the elements to allow easier e2e testing

File: dna-frontend/src/app/core/components/sidebar/sidebar.component.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
+a {
```
Comment (m-a-s-h-e-d) [inline dna-frontend/src/app/core/components/sidebar/sidebar.component.scss:1]: Same thing here
- Modify to classes instead of element tag

**Review by m-a-s-h-e-d — CHANGES_REQUESTED** [review]
A few comments on files

File: dna-frontend/src/app/core/components/sidebar/sidebar.component.html (hunk 1 above)
Comment (ebauss) [reply dna-frontend/src/app/core/components/sidebar/sidebar.component.html:1]: made those changes

File: dna-frontend/src/app/core/components/sidebar/sidebar.component.scss (hunk 2 above)
Comment (ebauss) [reply dna-frontend/src/app/core/components/sidebar/sidebar.component.scss]: made those changes

**Review by m-a-s-h-e-d — APPROVED** [review]

**Pull request merged**